net/https-dns-proxy/files/https_dns_proxy.config

```diff
@@ -1,8 +1,19 @@
 config https_dns_proxy
 	option listen_addr '127.0.0.1'
```

Comment:
It looks like you're setting a default for this below (and port) so these should't be needed. 
Does this just appear in the config as an example? Should it be commented out and documented?

Reply:
...and user/group.

Reply:
Example. I'd rather leave these both in the config file than create a README with the full description of options (which very few people will check anyways).

Reply:
OK